src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator started its work");

```

Comment:
The message should indicate what type of Machine is working right now. Consider making the message more informative by specifying the type of Machine, such as 'Excavator started its work'.